### PR TITLE
Removed Web-SDK-for-plugins document

### PR DESCRIPTION
**Pull request by JoshuaAdemola**

### CODE REVIEW

File: content/users-guide/cockpit-bundle/managing-assets.md [hunk 1]
```diff
@@ -111,7 +111,7 @@ Devices show the following tabs:
 
 If dashboards have been created for a group or device, they will also be added as a tab. See [Working with Dashboards](#dashboards) for details.
 
-Moreover, additional tabs may be displayed here in case the application has been extended with plugins. See [Web SDK for plugins](/web-sdk-for-plugins/overview/) for details.
+Moreover, additional tabs may be displayed here in case the application has been extended with plugins. **See Web SDK for plugins for details.
```
Comment (BeateRixen) [inline content/users-guide/cockpit-bundle/managing-assets.md:114]: @janhommes If we remove the Web SDK for plugins documentation, where should this link point to?

Comment (janhommes) [reply content/users-guide/cockpit-bundle/managing-assets.md:114]: This is kind of outdated but still correct. Should point to the new SDK docs. I would point it to the tutorial: 
https://cumulocity.com/guides/web/tutorials/#add-a-tab-to-a-device

Saying something like "... has been extended with a custom Web SDK extension. See how to add a custom tab in our tutorials."
?

Comment (BeateRixen) [reply content/users-guide/cockpit-bundle/managing-assets.md:114]: @JoshuaAdemola Could you please resolve this comment?

File: content/benutzerhandbuch/cockpit-de-bundle/managing-assets.md [hunk 2]
```diff
@@ -100,7 +100,7 @@ Bei Geräten werden die folgenden Registerkarten angezeigt:
 
 Wenn für eine Gruppe oder ein Gerät Dashboards erstellt wurden, werden diese ebenfalls als Registerkarten angezeigt. Weitere Informationen finden Sie unter [Verwenden von Dashboards](#dashboards).
 
-Es können zusätzliche Registerkarten angezeigt werden, falls die Anwendung mit Plugins erweitert wurde. Siehe auch [Web SDK for plugins](/web-sdk-for-plugins/overview/).
+Es können zusätzliche Registerkarten angezeigt werden, falls die Anwendung mit Plugins erweitert wurde. **Siehe auch Web SDK for plugins.
```
Comment (BeateRixen) [inline content/benutzerhandbuch/cockpit-de-bundle/managing-assets.md:103]:
```suggestion
Es können zusätzliche Registerkarten angezeigt werden, falls die Anwendung mit einer eigenen Web SDK-Extension erweitert wurde. Siehe auch [Web SDK tutorials](/web/tutorials/#add-a-tab-to-a-device) für Informationen zum Hinzufügen einer eigenen Registerkarte.
```

**Review by BeateRixen — APPROVED** [review]

**Review by rssag — APPROVED** [review]

**Pull request merged**